src/components/Layout/RightPaneLayout/RightPaneLayout.tsx

```diff
@@ -0,0 +1,52 @@
+import { AnimatePresence, motion } from "framer-motion";
+import React, { ReactElement } from "react";
+import { Css, Palette, px } from "../../../Css";
+import { useRightPaneContext } from "./RightPaneContext";
+
+export function RightPaneLayout({ children, paneBgColor }: { children: ReactElement; paneBgColor?: Palette }) {
+  const { isRightPaneOpen, rightPaneContent } = useRightPaneContext();
+
+  // Todo pass as prop?
+  const paneWidth = 450;
+
+  return (
+    <div css={Css.h100.df.$}>
+      <>
+        <motion.div
+          layout="position"
+          key="rightPaneLayoutPageContent"
+          css={Css.h100.if(!!isRightPaneOpen).overflowX("scroll").mr3.if(!isRightPaneOpen).overflowX("unset").$}
```

Comment:
Can we use `overflowXAuto` instead of `scroll`? The benefit is not creating a scrollbar gutter when unnecessary.

Reply:
updated to use `overflowXAuto`